website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,47 @@
+
+With the above changes Jest 26 is now 4 MiB smaller than Jest 25.5.4 (53 → 49 MiB). Please keep in mind that many dependencies like Babel are likely already part of your project. Jest's own size was reduced by 1.2 MiB (4.3 -> 3.1 MiB).
+
+While this is a good start, it isn’t enough to make Jest meaningfully leaner. We are planning on gradually reducing Jest's and it's dependency tree size by about 70%. Most of the changes to reduce Jest’s default setup will be small breaking changes and we want to bring the community with us as much as possible. Therefore we will make the following changes in Jest 27 and Jest 28 using a [“Tick-Tock" release process](https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model):
+
+* Jest 27 will ship with a new test runner (`jest-circus`), new fake timers with Lolex and the node.js environment by default. `jest-jasmine2`, the previous FakeTimers implementation and `jsdom` will still be bundled so users can keep using them by changing one-line each in the configuration.
+* Jest 28 will remove `jest-jasmine2`, FakeTimers and `jsdom` from the default distribution of Jest. The packages will still be actively maintained as part of the Jest project and be published separately. Users will need to install these packages to use them.
+
+Upon upgrading to these major releases, Jest will guide you through the necessary changes. If you’d like to get ahead and migrate to the new architecture now you can upgrade to Jest 26 and add the following configuration options:
+
+```
+TODO Simen/jeysal: configuration example
+```
+
+## Other Breaking Changes in Jest 26
+
+* Dropped support for Node 8 ([#9423](https://github.com/facebook/jest/pull/9423))
+* `[jest-environment-jsdom]` Upgrade `jsdom` to v16 ([#9606](https://github.com/facebook/jest/pull/9606))
+* `[jest-runtime]` Remove long-deprecated `require.requireActual` and `require.requireMock` methods ([#9854](https://github.com/facebook/jest/pull/9854))
+* `[jest-haste-map]` Removed `providesModuleNodeModules` ([#8535](https://github.com/facebook/jest/pull/8535))
+* `[jest-circus]` Fail tests if a test takes a done callback and have return values ([#9129](https://github.com/facebook/jest/pull/9129))
+* `[jest-circus]` Throw a proper error if a test / hooks is defined asynchronously ([#8096](https://github.com/facebook/jest/pull/8096))
+* ***TODO***
+
+## Stay Safe
+
+We are all currently experiencing an unprecedented time of uncertainty. If you are struggling financially, we would like to use [Jest’s OpenCollective fund](https://opencollective.com/jest) to help new and existing contributors. We have an existing [list of issues with a bounty](https://github.com/facebook/jest/issues?q=is%3Aissue+is%3Aopen+bounty+label%3A%22Has+Bounty%22) but are open to offering a bounty on any of our current open issues - you can mention that this issue should have a bounty in the issue or contact us privately via _x_.
```

Comment:
What's the best way to reach us privately? :D

Reply:
DM you on twitter? 😀

Reply:
Yeah I'm fine with that.